crates/sui-indexer/src/processors/address_processor.rs

```diff
@@ -1,12 +1,20 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use tracing::info;
+use tracing::{info, warn};
+
+use sui_json_rpc_types::{
+    CheckpointId, SuiTransactionBlockEvents, SuiTransactionBlockResponseOptions,
+};
 
 use crate::errors::IndexerError;
+use crate::models::addresses::{dedup_from_addresses, dedup_from_and_to_addresses};
 use crate::store::IndexerStore;
+use crate::types::{AddressData, CheckpointTransactionBlockResponse};
 
-const ADDRESS_STATS_BATCH_SIZE: i64 = 100;
+const ADDRESS_STATS_BATCH_SIZE: usize = 100;
```

Comment:
Why switch this from i64 to usize if you just cast it later anyways?

Reply:
const size of `usize` type is usually safer and thus preferred here, not a big deal here though